src/compat/dbval.h

```diff
@@ -535,6 +535,7 @@
 
 #define db_make_db_char(v, c, coll, p, s) \
     ((v)->data.ch.info.style = MEDIUM_STRING, \
+     (v)->data.ch.info.is_max_string = false, \
      (v)->data.ch.medium.codeset = (c), \
      (v)->data.ch.medium.size = (s), \
      (v)->data.ch.medium.buf = (char *) (p), \
```

Comment:
@lucianlazar2211 

Would you please also change `db_make_string`, `db_make_char`, `db_make_varchar`, `db_make_nchar` and `db_make_varnchar` of the file?